travis/js_tests.sh

```diff
@@ -16,11 +16,11 @@ function run_test {
     return $status
 }
 
-run_test npm run codecov
-run_test npm run lint
-run_test npm run fmt:check
-run_test npm run scss_lint
-run_test npm run flow
-run_test node node_modules/webpack/bin/webpack.js --config webpack.config.prod.js --bail
+#run_test npm run codecov
+#run_test npm run lint
+#run_test npm run fmt:check
+#run_test npm run scss_lint
+#run_test npm run flow
+#run_test node node_modules/webpack/bin/webpack.js --config webpack.config.prod.js --bail
```

Comment:
Why are these commented out? Ditto for `python_tests.sh` below

Reply:
Whoops! Had that in there to speed up deployment